grapher/dataTable/DataTable.scss

```diff
@@ -109,18 +149,11 @@
     }
 
     tbody tr:nth-child(2n) td {
-        background-color: rgb(250, 250, 250);
-    }
-    tbody tr:nth-child(2n) td.sorted {
-        background-color: rgb(246, 246, 246);
-    }
-
-    td.sorted {
-        background-color: rgb(250, 250, 250);
+        background-color: rgba(250, 250, 250);
```

Comment:
I find this background color to be a little too faint, even with good eyes it is barely noticeable.
Maybe make it a teeny tiny bit darker, and the hover color too so we still have some separation?

Reply:
darkened!